lifetiles-sequence/src/main/java/nl/tudelft/lifetiles/sequence/model/SequenceSegment.java

```diff
@@ -354,7 +364,7 @@ public Mutation determineMutation() {
      * @return the interestingness score
      */
     public double interestingness() {
-        if (content.isEmpty()) {
+        if (content.isEmpty() || content instanceof SegmentStringCollapsed) {
```

Comment:
You turn every segment into SegmentStringCollapsed, and then you say that if they are SegmentStringCollapsed then they have 0 interestingness. Sooo everything has interestingness 0 now?